packages/did/src/utils/parser.ts

```diff
@@ -0,0 +1,34 @@
+export const didPattern = `^did:[a-z0-9]+?:?[a-z0-9]+?:(0x[A-Fa-f0-9]{40})$`;
+
+/**
+* Checks if did is valid and returns the did method
+*
+* @param did
+* @private
```

Comment:
Why `@private` here? ts-doc says that it is discouraged as it may be removed: https://typedoc.org/guides/doccomments/#%40public%2C-%40protected%2C-and-%40private

Reply:
Agreed, `private` annotations were used in [crypto.ts](https://github.com/energywebfoundation/ew-did-registry/blob/eb2ebbc922a6aad792018a1db14c8230de190748/packages/did-ethr-resolver/src/utils/crypto.ts#L52) so thought of following the same pattern, I will remove it from everywhere.